src/QsCompiler/TestProjects/test15/test15.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.Quantum.Sdk/0.19.2109165653">
+﻿<Project Sdk="Microsoft.Quantum.Sdk/0.27.244707">
```

Comment:
The above projects are being changed to Quantum SDK "0.27.253010". Why is this being changed to "0.27.244707"?

Reply:
Isn't this just an update to a more up-to-date sdk version?

Reply:
Yes, but per the comment, why not to the same version as the other projects got updated to?

Reply:
There was a specific commit regarding this particular change. I don't remember exact details. 
Let me try to unify the version to 0.27.253010. If it works then I'll update the PR.
Thanks for catching.

It was something related to the test that intentionally tests some older functionality. Presumably in [this line](https://github.com/microsoft/qsharp-compiler/blob/9bfde74871358d273198d3752fe466e79c779a0e/src/QsCompiler/Tests.LanguageServer/Tests.cs#L403) and [this fragment](https://github.com/microsoft/qsharp-compiler/blob/9bfde74871358d273198d3752fe466e79c779a0e/src/QsCompiler/Tests.LanguageServer/Tests.cs#L377-L400).

Reply:
Fixed.